### DISCUSSION TITLE
Disagreement over "persistent" default between dhcpcd.conf and man page

### DISCUSSION THREAD

**Discussion by GrantEdwards** [rsmarples/dhcpcd#140] [General]
This is trivial, but it still took me a while to figure it out...

The persistent option is _enabled_ by default in the dhcpcd.conf file.

The man page says that it's "normally" _disabled_. I interpreted that to mean that I had to take some action to enable it (either pass the -p option or edit the .conf file).

[I was trying to figure out why there was obsolete, incorrect info in my resolv.conf file.]

Perhaps either change the man page to state that persistent is enabled by the default dhcpcd.conf file or change the .conf file to match the "normally disabled" description in the man page? [My personal preference would be the latter.]

**Comment by ido** [comment]
Clarified the wording in the man page in commit  4a3be174.
